gamedb.yaml

```diff
@@ -261,12 +326,30 @@
   banner: steam:290510
   status: verified
 
+- name: Gods Will Fall
+  platform: epic-store
+  id: 69911638c74449d2a2651241be1b3ad4
+  banner: steam:1243690
+  status: unsupported
+
+- name: Gone Home
+  platform: epic-store
+  id: Flier
+  banner: steam:232430
+  status: unsupported
+
 - name: GoNNER
   platform: epic-store
   id: Emperor
   banner: steam:437570
   status: verified
 
+- name: Guild of Dungeoneering
+  platform: epic-store
+  id: b8bdc716e79e461bbf161d99f842f29a
+  banner: steam:317820
```

Comment:
This is the ultimate edition on Steam, is the version on EGS here also the ultimate edition?

Reply:
it's a standard edition game